fsspec_reference_maker/combine.py

```diff
@@ -186,7 +186,12 @@ def _build_output(self, ds, ds0, fss):
             accum[accum_dim] = [np.array(a, dtype="M8") for a in accum[accum_dim]]
             attr.pop('units')
             attr.pop('calendar')
-        acc = np.concatenate(accum[accum_dim]).squeeze()
+    
+        if np.array(accum[accum_dim]).ndim == 1:
```

Comment:
Perhaps use [`atleast_1d`](https://numpy.org/doc/stable/reference/generated/numpy.atleast_1d.html) ?

Reply:
Perhaps like this?
```
acc = np.concatenate(np.atleast_1d(a) for a in accum[accum_dim]).squeeze()
```

Reply:
That seems to work great, I'll update the PR